tests/providers/aws/services/securityhub/securityhub_service_test.py

```diff
@@ -72,13 +99,21 @@ def test__get_session__(self):
 
     def test__describe_hub__(self):
         # Set partition for the service
-        current_audit_info.audited_partition = "aws"
-        securityhub = SecurityHub(current_audit_info)
-        assert len(securityhub.securityhubs) == 1
-        assert (
-            securityhub.securityhubs[0].arn
-            == "arn:aws:securityhub:us-east-1:0123456789012:hub/default"
-        )
-        assert securityhub.securityhubs[0].id == "default"
-        assert securityhub.securityhubs[0].standards == "cis-aws-foundations-benchmark "
-        assert securityhub.securityhubs[0].integrations == "prowler "
+        current_audit_info = self.set_mocked_audit_info()
+
+        with patch(
+            "prowler.providers.aws.lib.audit_info.audit_info.current_audit_info",
+            new=current_audit_info,
+        ):
```

Comment:
You don't need this here since the `current_audit_info` is not being imported anywhere.